### ISSUE TITLE
Inconsistent results when using `FETCH` with `boundary.post`

### ISSUE THREAD

**Issue by tmacro** [senx/warp10-platform#903]
When using the HTTP API to ingest events the first class ingested is ignored by `FETCH` when specifying  `boundary.post`.

For example: if an event is written for both `classA` and `classB` in sequence and the timestamp for `classA` is < `classB` the following `FETCH` should return the `classA` event, instead it returns the `classB` event. If only a single class is present no results are returned by `FETCH`

```
{
  'token' $rt
  'class' '~.*'
  'labels' {}
  'end' MINLONG 1 +
  'count' 0
  'boundary.post' 1
} FETCH
```

I've written a script to show this in action here: https://gist.github.com/tmacro/90d328a9743f50b2c21bce71a11c80a9

**Comment by ftence** [comment]
Thanks for this detailed issue.

We could reproduce the problem and a fix is on the way: #902

**Comment by hbs** [comment]
The issue was related to two things which were addressed in PR #902.

The first one was a case not taken into account when post boundaries were requested but when this was actually fixed we encountered a bug in the underlying LevelDB library which caused iterators to become invalid when seeking past the last key in the key space (which is the case you encountered when starting from an empty DB). To address this issue we forced the creation of a dummy record with a key after the last raw data key. This record is added once and for all when LevelDB is first open and fixes the issue since no iterator on the raw data key space can now become invalid.
